backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/navigationtabgroup/internal/BpkNavigationTabGroupImpl.kt

```diff
@@ -50,12 +53,25 @@ internal fun BpkNavigationTabGroupImpl(
         horizontalArrangement = Arrangement.spacedBy(BpkSpacing.Sm),
     ) {
         itemsIndexed(items = tabs) { index, tab ->
-            NavigationTabItem(
-                tab = tab,
-                selected = index == selectedIndex,
-                style = style,
-            ) {
-                onItemClicked.invoke(tabs[index])
+            if (behaviouralEventWrapper != null) {
+                behaviouralEventWrapper(tab, Modifier) {
+                    NavigationTabItem(
```

Comment:
tiny nit: you won't have to pass the `clickHandleScope` to the `NavigationTabItem`, but can just invoke the callback in the item click callback here